pyspades/vxl_c.cpp

```diff
@@ -149,7 +149,7 @@ int check_node(int x, int y, int z, MapData * map, int destroy)
         z = current_node->z;
         if (z >= 62) {
             marked.clear();
-            return 1;
+            return 0;

```

Comment:
This is related to the `check_point` function returning a count instead of bool. It's used in the `destroy_point` function where its return value is used to increment the count of blocks destroyed. Looks fine to me.